app/server/app-insights.ts

```diff
@@ -7,6 +7,7 @@ export const enable = () => {
 };
 
 export const trackException = exception => {
-  console.log(exception)
+  // tslint:disable-next-line: no-console
+  console.log(exception);
```

Comment:
I accidentally pushed this in.  Though something to log when running locally would be good.